### DISCUSSION TITLE
Chakra doesn't display react-split-pane

### DISCUSSION THREAD

**Discussion by somguyth** [chakra-ui/chakra-ui#4381] [Q&A]
I'm using `chakra-ui`, along with `react-split-pane`. 

If I use one *or* the other, it'll work well. If I use them *together*, the page is blank with no console error message. Here's three examples.

Using them together:

```
// Doesn't work, page is blank w/ no error message.

import React from "react";
import ReactDOM from "react-dom";
import { ChakraProvider } from "@chakra-ui/react"
import SplitPane from "react-split-pane";

export function Test() {
    return(
        <ChakraProvider>
            <SplitPane split="vertical" minSize={200} defaultSize={200} maxSize={400}>
                <div>123</div>
                <div>456</div>
            </SplitPane>
        </ChakraProvider>
    )
}

ReactDOM.render(
    <Test />,
    document.getElementById("root")
);
```

Using just `react-split-pane`:
```
// Works perfectly

export function Test() {
    return(
         <SplitPane split="vertical" minSize={200} defaultSize={200} maxSize={400}>
             <div>123</div>
             <div>456</div>
         </SplitPane>
    )
}
```

Using just `chakra-ui`:
```
// Works perfectly

export function Test() {
    return(
        <ChakraProvider>
             <div>123</div>
             <div>456</div>
        </ChakraProvider>
    )
}
```

Any ideas as to what could be happening here?

I'm using React+Typescript with webpack.

**Comment by MikeWillis** [comment]
It seems to be working for me...? https://codesandbox.io/s/zen-silence-4dpkq?file=/src/index.js

Not sure about why the dividers overrun the container divs, I'm not familiar with the split pane plugin, but in any case it seems to be working

**Reply by somguyth** [reply]
> It seems to be working for me...? https://codesandbox.io/s/zen-silence-4dpkq?file=/src/index.js
> 
> Not sure about why the dividers overrun the container divs, I'm not familiar with the split pane plugin, but in any case it seems to be working

Here's a more basic example (same code as in my OP) which doesn't work: https://codesandbox.io/s/patient-worker-x65r5?file=/src/index.js

It seems the absence of a `css` file leads to the output being totally blank whenever I have both chakra and react-split-pane running?

**Reply by MikeWillis** [reply]
yes I see the same issue, I think there's a conflict somewhere between the source code rendered by the split pane component, and Chakra UI's styles.

If you look at the rendered source code on the example you linked, the HTML is there for each pane (the divs are rendered under `#rootA`). That's why there's no javascript error showing in console - there's nothing wrong on that side. If you remove the `ChakraProvider` container, the content becomes visible (but split pane still isn't functional, I don't know why, I guess the split pane plugin needs some css or something).

So I suppose you have no choice but to do something along the lines of:

1. dig further to figure out exactly where Chakra's styles aren't agreeing with the HTML
2. either modify / override Chakra's theme, or modify / override the split pane css
